src/lib/coaler/io/OutputWriter.cpp

```diff
@@ -24,4 +42,31 @@ namespace coaler::io {
             sdf_writer->write(entry.getMolecule(), pose_id);
         }
     }
+
+    /*----------------------------------------------------------------------------------------------------------------*/
+
+    void OutputWriter::writeConformersToSDF(const std::string &folder_path,
+                                            const std::vector<RDKit::ROMOL_SPTR> &mols) {
+        std::string file_path = folder_path;
+        if (file_path.back() != '/') {
+            file_path += "/";
+        }
+        unsigned magnitude = std::to_string(mols.size()).length();
+        for (unsigned molId = 0; molId < mols.size(); molId++) {
+            auto mol = mols.at(molId);
+
+            std::string currentFilePath = file_path + "mol_" + get_formatted_string(molId, magnitude) + ".sdf";
+            std::ofstream output_file(currentFilePath);
+
+            if (!output_file.is_open()) {
```

Comment:
Jetzt kann er nur schreiben wenn der Ordner schon vorhanden ist. Wäre gut, wenn er den Ordner auch neu erstellen könnte.

Reply:
not sure. angenommen du hast nen typo in deinem pfad, und auf einmal hast du ein /tmü ordner auf deinem root liegen. Finde es okay wenn man sich ggf wie in jedem anderen tool auch im zweifelsfall fix einen ordner anlegen muss. Dann bekommst du auch kein problem mit folder chmods und so

Reply:
True, daran habe ich nicht gedacht, finde das halt selber immer bisschen nervig. Aber better safe than sorry